cli/pcluster/resources/batch/docker/scripts/entrypoint.sh

```diff
@@ -18,6 +18,11 @@ if [[ ! -z "${EFS_FS_ID}" ]] && [[ ! -z "${AWS_REGION}" ]] && [[ "${EFS_SHARED_D
   /parallelcluster/bin/mount_efs.sh "${EFS_FS_ID}" "${AWS_REGION}" "${EFS_SHARED_DIR}"
 fi
 
+# mount RAID via nfs
+if [ ! -z ${RAID_SHARED_DIR} ]; then
```

Comment:
If there is always going to be a RAID_SHARED_DIR? If the user does not request RAID, will there be RAID_SHARED_DIR?